drivers/storage/portworx/components.go

```diff
@@ -714,6 +779,52 @@ func (p *portworx) createCSIDriver(
 	)
 }
 
+func (p *portworx) createAutopilotConfigMap(
+	t *template,
+	ownerRef *metav1.OwnerReference,
+) error {
+	config := "providers:"
+	for _, provider := range t.cluster.Spec.Autopilot.Providers {
+		keys := make([]string, 0, len(provider.Params))
+		for k := range provider.Params {
+			keys = append(keys, k)
+		}
+		sort.Strings(keys)
```

Comment:
why sort ?

Reply:
Helps in comparing the params and see if anything has changed.